netebpfext/net_ebpf_ext.c

```diff
@@ -195,6 +253,46 @@ static net_ebpf_ext_wfp_callout_state_t _net_ebpf_ext_wfp_callout_state[] = {
         L"Resource Release callout driver for eBPF",
         FWP_ACTION_CALLOUT_TERMINATING,
     },
+    {
+        &EBPF_HOOK_FLOW_ESTABLISHED_CALLOUT_V4,
+        &FWPM_LAYER_ALE_FLOW_ESTABLISHED_V4,
+        _net_ebpf_ext_flow_established_classify,
+        _net_ebpf_ext_no_op_notify,
+        _net_ebpf_ext_no_op_flow_delete,
+        L"Flow Established V4 Callout",
+        L"Flow Established callout driver for eBPF",
+        FWP_ACTION_CALLOUT_TERMINATING,
+    },
+    {
+        &EBPF_HOOK_FLOW_ESTABLISHED_CALLOUT_V6,
+        &FWPM_LAYER_ALE_FLOW_ESTABLISHED_V6,
+        _net_ebpf_ext_flow_established_classify,
+        _net_ebpf_ext_no_op_notify,
+        _net_ebpf_ext_flow_delete,
```

Comment:
we don't want to know when the flows are deleted?

Reply:
We do, that will be implemented in the _net_ebpf_ext_flow_delete function. Will be implementing it afterwards. I only have the skeleton for the callouts right now.